assets/multiqc_config.yaml

```diff
@@ -0,0 +1,13 @@
+module_order:    
```

Comment:
@MarieLataretu oh interesting. What is the rationale behind this `multiqc_config.yaml`? You are using this to create a channel?

Reply:
ahhh, and then you use this as a config file in the multiQC module, correct? nice

Reply:
right, I'm also working on adding results from our custom modules (TPM filter and DESeq2) - that goes all the to MultiQC config